Add clarification as to the purpose of Metrics endpoint.

travelton opened the pull request:
Added paragraph explaining purpose of Metrics endpoint, along with suggested solution for capturing usage data.

travelton committed:
Add clarification as to the purpose of Metrics endpoint.

Added paragraph explaining purpose of Metrics endpoint, along with
suggested solution for capturing usage data.

ptgott commented on docs/pages/reference/metrics.mdx:
```diff
@@ -2,12 +2,38 @@
 
 </Notice>
 
+Teleport metrics data is primarily used for performance monitoring. If you'd like to
```
Does "primarily" mean that there is another use? Should we say "Teleport metrics are intended for performance monitoring"?

travelton replied:
> "Teleport metrics are intended for performance monitoring" 

Correct... It's intended to be used for performance monitoring, but you could be using it for any number of things. Let's go with this. I'll adjust the PR.

ptgott commented on docs/pages/reference/metrics.mdx:
```diff
@@ -2,12 +2,38 @@
 </Notice>
 
+Teleport metrics data is primarily used for performance monitoring. If you'd like to
+monitor Teleport usage, consider utilizing our *fluentd* plugin to push Audit Events into your
```
```suggestion
monitor Teleport usage, consider utilizing our Event Handler plugin to push Audit Events into your
```
While the Event Handler was designed with Fluentd in mind, I think we should use a more general title so we don't put off users who are using Logstash, a Splunk forwarder, or another solution to forward audit logs, since the Event Handler can work with these, too (albeit with some extra steps, e.g., in the [Elastic Stack guide](https://goteleport.com/docs/management/guides/elastic-stack/)).

ptgott commented on docs/pages/reference/metrics.mdx:
```diff
@@ -2,12 +2,38 @@
 
+Teleport metrics data is primarily used for performance monitoring. If you'd like to
+monitor Teleport usage, consider utilizing our *fluentd* plugin to push Audit Events into your
+preferred logging aggregation system (Elastic, Splunk, Sumo, etc). 
```
```suggestion
preferred logging aggregation system (Elastic, Splunk, Sumo Logic, etc). 
```

ptgott commented on docs/pages/reference/metrics.mdx:
```diff
@@ -2,12 +2,38 @@
+monitor Teleport usage, consider utilizing our *fluentd* plugin to push Audit Events into your
+preferred logging aggregation system (Elastic, Splunk, Sumo, etc). 
+
+<ScopedBlock scope="oss">
```
These three `ScopedBlock`s are the same for all scopes. Should we collapse them into one bulleted list without the `ScopedBlock` tag?

travelton replied:
New to this... I was trying to align with the implementations I saw on other pages. It seems the ScopedBlock adds the appropriate URL query string, depending on the current scope. i.e. If I'm on OSS, clicking on the enclosed links would keep me on the OSS scoped docs. Let me test this out a bit more.

travelton replied:
Ah! The URL query params follow you regardless. Cool, I can remove this.

travelton committed:
PR Feedback.

1. Adjusted verbiage.
2. Removed ScopedBlocks.
3. Linked to Elastic instructions.

travelton commented:
@ptgott I added a link to the article you mentioned, `Monitor Teleport Audit Events with the Elastic Stack`. Happy to remove it, but felt right linking there.

travelton committed:
Merge branch 'master' into travis/docs-metrics-note
